lib/ansiblelint/runner.py

```diff
@@ -3,9 +3,10 @@
 import os
 from typing import List, Set
 
-import ansiblelint.utils
 import ansiblelint.file_utils
 import ansiblelint.skip_utils
+import ansiblelint.utils
+
```

Comment:
This sort of separation is probably undesirable because both parts essentially represent the same thing.
```suggestion
```